hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestDataUtil.java

```diff
@@ -104,6 +105,14 @@ public static OzoneVolume createVolume(OzoneClient client,
 
   }
 
+  public static byte[] createKey(OzoneBucket bucket,
+                                 String keyName, int length) throws IOException {
+    byte[] content = RandomStringUtils.random(length, 0, 0,
+        true, true, null, new SecureRandom()).getBytes(UTF_8);
```

Comment:
- Let's clarify that this method generates string content by renaming to `createStringKey`.
- Please avoid `new SecureRandom()` or `new Random()` for single use.  Replace with `RandomStringUtils.secure().nextAlphanumeric(length).getBytes(UTF_8)`.
- nit: Please do not format method signature like this. Whenever visibility / return type / method name / other modifiers are changed, we would have to reindent all parameters.